Resources/Prototypes/Alerts/alerts.yml

```diff
@@ -39,7 +39,7 @@
   icon: /Textures/Interface/Alerts/Pressure/highpressure.png
   maxSeverity: 2
   name: "[color=red]High Pressure[/color]"
-  description: "The air around you is [color=red]hazardously thick[/color]. A [color=green]fire suit[/color] would protect you."
+  description: "The air around you is [color=red]hazardously thick[/color]. A [color=green]fire suit[/color] would be enough protect you."
```

Comment:
> "A pressurized suit would be enough to protect you."

Reply:
Difference is that at this point a firesuit would be enough to protect you, the harsher atmosphere conditions have a "Space suit"